components/odh-notebook-controller/controllers/notebook_webhook.go

```diff
@@ -376,9 +377,11 @@ func certValidator(cm *corev1.ConfigMap, dataKey string, log logr.Logger) error
 		// Attempt to parse the certificate
 		_, err := x509.ParseCertificate(odhBlock.Bytes)
 		if err != nil {
-			return fmt.Errorf("error parsing certificate for key '%s' in ConfigMap odh-trusted-ca-bundle: %v", dataKey, err)
+			log.Error(err, fmt.Sprintf("error parsing certificate for key '%s' in ConfigMap odh-trusted-ca-bundle", dataKey))
+			return nil
 		}
 	} else if len(odhCertData) > 0 {
+		log.Error(nil, fmt.Sprintf("invalid certificate format for key '%s' in ConfigMap odh-trusted-ca-bundle", dataKey))
```

Comment:
Hm, does this mean that we log this error here and then also again where the `certValidator` function is used and checked for the return error (lines 313 and 328)?

`odh-trusted-ca-bundle` - would deserve some constant, but that's just nitpick